### ISSUE TITLE
shellcheck.net is bugging out! Double pasting, no search for text feature working and can't resize the window like I normally can.

### ISSUE THREAD

**Issue by slyfox1186** [koalaman/shellcheck#2870]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/your/shebang
Your script
```

#### Here's what shellcheck currently says:
```
Reports nothing. No output given.
```
#### Here's what I wanted or expected to see:

I expected my script to normally be parsed and have the feed back on what I am doing incorrectly. I wanted it to work like it always does.

Every time I copy and paste from my script editor into the web gui at shellcheck.net it double pastes the code and I also can't resize the window like I normally can, and I can't use any other feature like the search for text feature that is usually available.

**Comment by koalaman** [comment]
I'm not able to reproduce any of this. Which browser and OS is this? Have you tried with any browser extensions disabled?

**Comment by koalaman** [comment]
Please reopen if the issue persists.
